DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -445,5 +451,18 @@ def execute(self):
       log.error("Error deleting operations", res)
       return res
 
-
     return S_OK()
+
+  @staticmethod
+  def __sendAccounting(ftsJob):
+    """ prepare and send DataOperation to AccountingDB
+
+        :param ftsJob: the FTS3Job from which we send the accounting info
+    """
+
+    dataOp = DataOperation()
+    dataOp.setStartTime(fromString(ftsJob.submitTime))
+    dataOp.setEndTime(fromString(ftsJob.lastUpdate))
+
+    dataOp.setValuesFromDict(ftsJob.accountingDict)
+    dataOp.commit()
```

Comment:
Would it be better to use "delayeCommit" to reduce the amount of queries to the DataStoreClient?

Reply:
Can do ! I did not even know about that